ROS/Chapter_1.ipynb

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
<p>what about making the data available from the bambi repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I was debating this but not sure what to do. Here's factors I have</p><ul><li>Aki is still changing around the data in preparation for book</li><li>Why duplicate if its already being maintained somewhere else?</li></ul><p> </p><p>That being said not opposed maybe we should copy it here? </p>

Reply:
Yeah I think having a copy is a better option. We may want to do it, once the book is published. So we can merge this.

Reply:
Do we want to merge and then update when book is out? I'm fine with whatever, just want to know so I can create another PR with just the container whatnot so I'm not branching off of branches as I go through the chapters